shell/packages/sandstorm-email/email.js

```diff
@@ -23,8 +23,9 @@ var makePool = function (mailUrlString) {
   var auth = false;
   if (mailUrl.auth) {
     var parts = mailUrl.auth.split(':', 2);
-    auth = {user: parts[0] && decodeURIComponent(parts[0]),
-            pass: parts[1] && decodeURIComponent(parts[1])};
+    console.log(parts);
```

Comment:
Probably shouldn't keep logging the parts, particularly if they might contain credentials to a relay.